ui/src/app/pages/data/data-page.tsx

```diff
@@ -66,6 +66,7 @@ enum Tabs {
   SHOWALL = 'SHOW ALL',
   DATASETS = 'DATA SETS',
   COHORTS = 'COHORTS',
+  COHORTREVIEWS = 'COHORTREVIEWS',
```

Comment:
This should probably have a space in the string.

Reply:
Nice catch, done.